modules/node-scripts/bin.js

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env node
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {main} from './index.js';
+
+main().catch((error) => {
```

Comment:
You can change this to:

```javascript
try {
  await main();
}
catch(error) {
  ...
}
```

For better readibility. Apparently you can use top level await in node :tada: .